### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matsduf**
Update supported OS and perl

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,20 +45,31 @@ Supported operating system versions:
...
 * Ubuntu 14.04
 * Ubuntu 16.04
 
+_Currently, Debian 9 is not supported due to a known bug in Zonemaster::LDNS. That has to be resolved 
+before we can include Debian 9 as supported OS._
+
 Supported database engine versions:
 
 * MySQL 5.5, 5.7
 * PostgreSQL 9.3, 9.4, 9.5
```
Comment (mattias-p) [inline README.md]: Maybe update the supported databases too? I went ahead and compiled the eligible ones. Maybe we should ask Michal?

MySQL 5.5 (Debian 7.11, 8.9, FreeBSD, Ubuntu 14.04)
MySQL 5.6 (FreeBSD)
MySQL 5.7 (FreeBSD, Ubuntu 16.04)
MySQL 8.0 (FreeBSD)
MariaDB 5.5 (Centos 7) ?
Postgres 9.1 (Debian 7.11)
Postgres 9.2 (Centos 7, FreeBSD)
Postgres 9.3 (FreeBSD, Ubuntu 14.04)
Postgres 9.4 (Debian 8.9, FreeBSD)
Postgres 9.5 (FreeBSD, Ubuntu 16.04)
Postgres 9.6 (FreeBSD)
Postgres 10.0 (FreeBSD)

Comment (matsduf) [reply README.md]: We can only support MariaDB if it is really a plug-in replace for MySQL. At least an update of the installation documentation is needed.

Should we really support all those versions on FreeBSD? Maybe only MySQL 8.0 and PostgreSQL 10? Could be extended to 8.0+5.7 and 10.0+9.6, repectively?

File: README.md [hunk 2]
```diff
@@ -108,8 +119,16 @@ You can follow the project in these two mailinglists:
...
 "contact@zonemaster.net".
 
+
+
+
+ 
+
+
+
+
```
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md]: Yes. Room for your own notes. :-)

File: README.md [hunk 3]
```diff
@@ -45,20 +45,31 @@ Supported operating system versions:
...
 
 * MySQL 5.5, 5.7
 * PostgreSQL 9.3, 9.4, 9.5
 
 Supported Perl versions:
 
-* Perl 5.14, 5.16, 5.18, 5.20, 5.22
+Perl version | CentOS version | Debian version | Ubuntu version |FreeBSD version
+------------ | ---------------| -------------- | -------------- | --------------
+5.14         |        -       |        7       |         -      |        -
```
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md]: Will update.

**Comment by matsduf** [comment]
@mtoma, can you look at the selection of databases and versions?

File: README.md [hunk 4]
```diff
@@ -89,13 +99,13 @@ the source code.
 In order to install and run the different components, have a look at the
 installation documentation under each of the repositories below:
 
- * [net-ldns](https://github.com/dotse/net-ldns) - which is a fork of
-   of [ldns](https://www.nlnetlabs.nl/projects/ldns/). with a Perl frontend
- * [zonemaster-engine](https://github.com/dotse/zonemaster-engine) - which
+ * [zonemaster-ldns] - which is a fork of
```
Comment (mattias-p) [inline README.md]: Remove some line breaks in this section for readability in "raw mode".

File: README.md [hunk 5]
```diff
@@ -119,16 +129,28 @@ You can follow the project in these two mailinglists:
  * [zonemaster-users](http://lists.iis.se/cgi-bin/mailman/listinfo/zonemaster-users)
  * [zonemaster-devel](http://lists.iis.se/cgi-bin/mailman/listinfo/zonemaster-devel)
 
-### Contact or Bug reporting 
-
-For any contacts or bug reporting, please send a mail to
-"contact@zonemaster.net".
-
-
+If you create a pull request, please select the "develop" branch in the relevant
```
Comment (mattias-p) [inline README.md]: This sentence deserves a "Contributing" heading.

File: README.md [hunk 6]
```diff
@@ -53,23 +53,33 @@ Supported operating system versions:
...
-* PostgreSQL 9.3, 9.4, 9.5
+OS version   |  Supported MySQL version |  Supported Maria DB version  | Supported PostgreSQL
+-----------  | ------------------------ | ---------------------------- | --------------------
+CentOS 7     |                          |             5.5              |          9.2
+Debian 7.11  |           5.5            |                              |          9.1
+Debian 8.9   |           5.5            |                              |          9.4
+Ubuntu 14.04 |           5.5            |                              |          9.3
+Ubuntu 16.04 |           5.7            |                              |          9.5
+FreeBSD 10.3 |    5.5, 5.6, 5.7, 8.0    |                              |   9.2, 9.3, 9.4, 9.5, 9.6, 10.0       
+FreeBSD 11   |    5.5, 5.6, 5.7, 8.0    |                              |   9.2, 9.3, 9.4, 9.5, 9.6, 10.0
```
Comment (mattias-p) [inline README.md]: We should refer to FreeBSD 11.1 here.

File: README.md [hunk 7]
```diff
@@ -98,7 +115,7 @@ Zonemaster and the versions of the specific components. Be
 sure to read the release note of each component before installing or
 upgrading.
 
-### Participation
+## Participation
 
 The core development team are people from IIS and Afnic. However, you
 can submit code by forking this repository and create pull requests.
```
Comment (mattias-p) [inline README.md]: 'create pull requests' -> 'creating pull requests to the "develop" branch'

File: README.md [hunk 8]
```diff
@@ -32,43 +32,64 @@ for the zonemaster implementation are available.
 
 Also, in this repository, you can find a brief [user guide](USING.md).
 
-### Prerequisites
+## Prerequisites
 
 Zonemaster comes with documentation for and has been tested with these
```
Comment (mattias-p) [inline README.md]: This sentence looks funny now, since the following sentence has been changed into a heading.

File: README.md [hunk 9]
```diff
@@ -91,20 +91,27 @@ Zonemaster comes with localization for these locales:
 
 ## Repositories
```
Comment (mattias-p) [inline README.md]: I think the "Repositories" should be renamed "Components", and the language in the section contents be updated to describe things in terms of components.

File: README.md [hunk 10]
```diff
@@ -91,20 +91,27 @@ Zonemaster comes with localization for these locales:
...
  * [zonemaster-cli] - a Command Line Interface for the engine.
  * [zonemaster-backend] - which interfaces between the GUI and an API with the engine.
- * [zonemaster-gui](https://github.com/dotse/zonemaster-gui) - A graphical user
-   interface for the engine.
+ * [zonemaster-gui] - A graphical user interface for the engine.
+
+## Installation
+
+To install Zonemaster, start with installation of [zonemaster-engine] (which will
+draw in Zonemaster-LDNS) and then continue with the other parts. From the links above
```
Comment (mattias-p) [inline README.md]: "From the links above installation instructions you will find." -> "You will find installation instructions from the links above." :)

File: README.md [hunk 11]
```diff
@@ -91,20 +91,27 @@ Zonemaster comes with localization for these locales:
...
-installation documentation under each of the repositories below:
+The Zonemaster Product includes the following components:
 
  * [zonemaster-ldns] - which is a fork of of [ldns] with a Perl frontend.
  * [zonemaster-engine] - which contains the libraries doing the actual tests.
  * [zonemaster-cli] - a Command Line Interface for the engine.
  * [zonemaster-backend] - which interfaces between the GUI and an API with the engine.
- * [zonemaster-gui](https://github.com/dotse/zonemaster-gui) - A graphical user
-   interface for the engine.
+ * [zonemaster-gui] - A graphical user interface for the engine.
```
[the remark this answers is not in the document]
Comment (pawal) [reply README.md]: If fixing things:
* [zonemaster-ldns] - a fork of [ldns] with a Perl interface.
* [zonemaster-engine] - the Zonemaster test library.
* [zonemaster-cli] - the Zonemaster CLI.
* [zonemaster-backend] - a JSON/RPC interface for running tests.

File: README.md (hunk 9 above)
Comment (matsduf) [reply README.md]: Updated.

File: README.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md]: Updated. Not exactly as you suggested.

File: README.md (hunk 10 above)
Comment (matsduf) [reply README.md]: Updated.

**Comment by mattias-p** [comment]
I believe we should only support PostgreSQL 9.5 on FreeBSD. Backend depends on p5-DBD-Pg (latest is 3.7.0) which depends on postgresql95-client which seems to be rather picky about only being installed together with postgresql95-server.

**Pull request merged**